stripe/src/main/java/com/stripe/android/view/i18n/TranslatorManager.kt

```diff
@@ -0,0 +1,47 @@
+package com.stripe.android.view.i18n
+
+import android.app.Application
+import com.stripe.android.StripeError
+import com.stripe.android.view.i18n.TranslatorManager.setErrorMessageTranslator
+
+/**
+ * A class that provides a [ErrorMessageTranslator] for translating server-provided error
+ * messages, as defined in [Stripe API Errors Reference](https://stripe.com/docs/api/errors).
+ *
+ * See [com.stripe.android.view.PaymentMethodsActivity] for example usage.
+ *
+ * To use a custom [ErrorMessageTranslator] in your app, override [Application.onCreate] in
+ * your app's Application subclass and call [setErrorMessageTranslator].
+ *
+ * <pre>
+ * public class MyApp extends Application {
+ *   public void onCreate() {
+ *     super.onCreate();
+ *     TranslatorManager.setErrorMessageTranslator(new MyErrorMessageTranslator());
+ *   }
+ * }
+ * </pre>
+ */
+object TranslatorManager {
+    private val DEFAULT_ERROR_MESSAGE_TRANSLATOR = Default()
+
+    private var errorMessageTranslator: ErrorMessageTranslator? = null
```

Comment:
Any reasons to not make this non-nullable and assign it to `DEFAULT_ERROR_MESSAGE_TRANSLATOR` by default?

Reply:
Also you probably don't need the getter and setter, right? IIRC kotlin will generate those for you automatically when calling from Java

Reply:
I didn't want to change the signature, which allowed for a nullable `ErrorMessageTranslator`

I would have used a property, but I wanted the return type to be non-null, which I don't think is possible (i.e. setter = String?, getter = String)